File: PHP/Loop.php
#! /usr/bin/env php
<?php

print "\r\nこれからループ処理を試します\r\n\r\n";

print "\r\nfor (\$n=0; \$n<3; \$n++) {…}\r\n";
for ($n=0;$n<3;$n++) {
	print "n=${n} のループ\r\n";
}

print "\r\nforeach (array as \$c) {…}\r\n";
$cs=["壱","弐","参"];
foreach ($cs as $c) {
	print "c=\"${c}\" のループ\r\n";
}

print "\r\nwhile (~) {…}\r\n";
$str="";
while (strlen($str)<6) {
	$str.="*";
	print "$str\r\n";
}

print "\r\ndo {…} while (~)\r\n";
$str="";
do {
	$str.="+";
	print "$str\r\n";
} while (strlen($str)<6);

print "\r\n\r\n";

?>

File: Julia/Legacy/Loop.jl
#! /usr/bin/env julia

println("\r\nこれからループ処理を試します\r\n")

println("\r\nfor n = 0:2 …end\r\n")
for n = 0:2
	println("n=$(n) のループ")
end

println("\r\nfor n = 3.5:0.7:5 …end\r\n")
for n = 3.5:0.7:5
	println("n=$(n) のループ")
end

println("\r\nfor n = 0:2, m = n:2 …end\r\n")
for n = 0:2, m = n:2
	println("n=$(n) m=$(m) のループ")
end

println("\r\nfor c ∈ array …end\r\n") # ∈ の他に, in や = でも問題ない
cs=["壱","弐","参"]
for c ∈ cs
	println("c=\"$(c)\" のループ")
end

println("\r\nwhile ~ … end")
string=""
while length(string)<6
	global string
	string*="*"
	println(string)
end

println("\r\n")